src/test/java/teammates/test/cases/automated/FeedbackSessionEmailTaskQueueTest.java

```diff
@@ -213,7 +211,7 @@ public void testScheduleFeedbackSessionClosingEmails() throws Exception {
         int counter = 0;
         while (counter != 10) {
             FeedbackSessionsEmailTaskQueueCallback.resetTaskCount();
-            fsLogic.scheduleFeedbackSessionClosingEmails();
+            //fsLogic.scheduleFeedbackSessionClosingEmails();
```

Comment:
Yes, and in fact I have had the fix ready for some time, ready to be pushed along with the PR that will resolve #6243. It will be right after this one is merged.